Casks/w/wljs-notebook.rb

```diff
@@ -1,15 +1,37 @@
 cask "wljs-notebook" do
   arch arm: "arm64", intel: "x64"
 
-  version "2.5.5"
-  sha256 arm:   "4515cfa4b17f7b54361ebd45fb7093a7dbaaafa7ccb2b62a09cb42ecfee4f7f9",
-         intel: "051956a302fb0a97fc3584f8c7bafc0679bfed8ab77c8c54ecb181723248aa25"
+  version "2.5.6,2.5.6FIX"
+  sha256 arm:   "0aefd5dfe2f518c3a22e357193f196774e48232a4b7ac8bdeb58c48bcee25d33",
+         intel: "21f2e42bc6fff7b70e751dc6221a09bd7d613bedccb91b04a69e13ecc78e9377"
 
-  url "https://github.com/JerryI/wolfram-js-frontend/releases/download/#{version}/wljs-notebook-#{version}-#{arch}.dmg"
+  url "https://github.com/JerryI/wolfram-js-frontend/releases/download/#{version.csv.second || version.csv.first}/wljs-notebook-#{version.csv.first}-#{arch}.dmg"
   name "WLJS Notebook"
   desc "Javascript frontend for Wolfram Engine"
   homepage "https://github.com/JerryI/wolfram-js-frontend"
 
+  # The current version of wljs-notebook has a tag that differs
+  # from the version present in the filename
+  livecheck do
+    url :url
+    regex(/wljs[._-]notebook[._-]v?(\d+(?:\.\d+)+)/i)
+    strategy :github_releases do |json, regex|
+      json.map do |release|
+        next if release["draft"] || release["prerelease"]
+
+        tag_version = release["tag_name"]
```

Comment:
```suggestion
        tag_version = release["tag_name"][/^v?(\d+(?:\.\d+)+.*)$/i, 1]
```

This wouldn't work as expected if upstream starts using a tag format like `v1.2.3`, as the filename regex omits any leading `v` but the `tag_name` wouldn't (i.e., `1.2.3` != `v1.2.3`).